### ISSUE TITLE
[BUG] OTLP metrics error on export

### ISSUE THREAD

**Issue by joshuaphelpsms** [microsoft/promptflow#3341]
**Describe the bug**

Setting `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT` when running the Prompt flow server in a docker container produces the following error:

```sh
promptflow  | Invalid type <class 'NoneType'> of value None
promptflow  | Traceback (most recent call last):
promptflow  |   File "/opt/conda/envs/promptflow-serve/lib/python3.9/site-packages/opentelemetry/exporter/otlp/proto/common/_internal/__init__.py", line 110, in _encode_attributes
promptflow  |     pb2_attributes.append(_encode_key_value(key, value))
promptflow  |   File "/opt/conda/envs/promptflow-serve/lib/python3.9/site-packages/opentelemetry/exporter/otlp/proto/common/_internal/__init__.py", line 92, in _encode_key_value
promptflow  |     return PB2KeyValue(key=key, value=_encode_value(value))
promptflow  |   File "/opt/conda/envs/promptflow-serve/lib/python3.9/site-packages/opentelemetry/exporter/otlp/proto/common/_internal/__init__.py", line 88, in _encode_value
promptflow  |     raise Exception(f"Invalid type {type(value)} of value {value}")
```

**How To Reproduce the bug**

1. Run an OpenTelemetry collector
2. Run the chat basic prompt flow server with the metrics endpoint environment variable set `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT=http://${OTEL_COLLECTOR_HOST}:${OTEL_COLLECTOR_PORT_HTTP}/v1/metrics`
3. Send a message and wait for about 30 seconds
4. Check the prompt flow logs 

**Expected behavior**
I expect no error messages

**Running Information:**
 - Promptflow Package Version using `pf -v`:
{
  "promptflow": "1.11.0",
  "promptflow-core": "1.11.0",
  "promptflow-devkit": "1.11.0",
  "promptflow-tracing": "1.11.0"
}
- Linux - using docker.io/continuumio/miniconda3:latest container
- Python 3.9.16

This docker files for the container was created using the pf cli command `pf flow build --source <source_dir> --output <output_dir> --format docker`

**Additional context**
The metrics seem to make it to the OTEL collector so not sure what is failing

**Comment by wxpjimmy** [comment]
This has been fixed in the latest code. This is the PR: https://github.com/microsoft/promptflow/pull/3305/files

**Comment by brynn-code** [comment]
> Thanks for reporting the issue, this has been fixed in the latest code. This is the PR: https://github.com/microsoft/promptflow/pull/3305/files

Plus: This was expected to be included with promptflow 1.12.0.
